Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -245,7 +245,7 @@ class BrowserViewController: UIViewController {
     }
     
     // Display first ad when the user gets back to this controller if they havent seen one before
-    func displayMyFirstAdIfAvailable() {
+    func displayMyFirstAdIfAvailable(_ completion: (() -> Void)? = nil) {
```

Comment:
This change doesn't do anything? `completion` isn't called.

Either way, lets remove this change and simply copy/paste similar code into onboarding (barring figuring out how to show the my-first-ad URL).

Reply:
Fixed.